services/harborregistryctl/harborregistry.yml

```diff
@@ -0,0 +1 @@
+/Users/cdchris12/amazeeprojects/lagoon/services/harborregistry/harborregistry.yml
```

Comment:
what is this file for?

Reply:
`registry` and `registryctl` both use the same base config file. Resolved the softlink issue mentioned elsewhere, so this file should now be populated as expected.